crates/task-impls/src/quorum_proposal/handlers.rs

```diff
@@ -247,52 +243,6 @@ impl<TYPES: NodeType, V: Versions> HandleDepOutput for ProposalDependencyHandle<
 
     #[allow(clippy::no_effect_underscore_binding, clippy::too_many_lines)]
     async fn handle_dep_result(self, res: Self::Output) {
-        let high_qc_view_number = self.consensus.read().await.high_qc().view_number;
```

Comment:
This isn't really a comment on your code, but in `parent_leaf_and_state` (called from `publish_proposal`) we actually lookup by `high_qc`, I think it works, but I'd much rather see the parent view number be decided by the events we got rather than the high_qc